Week1/index.js

```diff
@@ -79,18 +79,16 @@
         id: 'list',
         class: 'contributor-list',
       });
-      let contributorURL;
-      let contributorItem;
-      let contributorData;
+      
       const list = document.getElementById('list');
       for (let i = 0; i < data.length; i++) {
-        contributorURL = createAndAppend('a', list, { href: data[i].html_url, target: '_blank' });
-        contributorItem = createAndAppend('li', contributorURL, { class: 'contributor-item' });
+        const contributorsURL = createAndAppend('a', list, { href: data[i].html_url, target: '_blank' });

```

Comment:
since you're going through a list of contributors you are handling a contributorUrl inside the loop, so the name here was totally fine before.

Reply:
I tried both ways and they both worked fine without conflicts.  Now home work is completed without problems.